ui/pages/confirmations/components/confirm/mmi-signature-mismatch-banner/mmi-signature-mismatch-banner.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { memo, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { MESSAGE_TYPE } from '../../../../../../shared/constants/app';
+import {
+  BackgroundColor,
+  BlockSize,
+  Display,
+  IconColor,
+  TextColor,
+  TextVariant,
+} from '../../../../../helpers/constants/design-system';
+import {
+  getAccountByAddress,
+  shortenAddress,
+} from '../../../../../helpers/utils/util';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import {
+  accountsWithSendEtherInfoSelector,
+  currentConfirmationSelector,
+  getSelectedInternalAccount,
+} from '../../../../../selectors';
+import {
+  Box,
+  Icon,
+  IconName,
+  Text,
+} from '../../../../../components/component-library';
+
+const MMISignatureMismatchBanner: React.FC = memo(() => {
+  const t = useI18nContext();
+  const currentConfirmation = useSelector(currentConfirmationSelector);
+  const selectedAccount = useSelector(getSelectedInternalAccount);
+  const allAccounts = useSelector(accountsWithSendEtherInfoSelector);
+
+  const fromAccount = useMemo(() => {
+    if (
+      !currentConfirmation ||
+      currentConfirmation.type !== MESSAGE_TYPE.PERSONAL_SIGN ||
```

Comment:
Just double-checking if this component is not shown for other signature types?

```js
  ETH_SIGN: 'eth_sign',
  ETH_SIGN_TYPED_DATA: 'eth_signTypedData',
  ETH_SIGN_TYPED_DATA_V3: 'eth_signTypedData_v3',
  ETH_SIGN_TYPED_DATA_V4: 'eth_signTypedData_v4',
```

Reply:
I found it there in personal sign, as we port other signatures to new designs we can enable this for other signatures also if they support it.